cn.apipost.apipost.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "cn.apipost.apipost",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "run.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--filesystem=home"
```

Comment:
why is home needed?

Reply:
the function requires users to import .json file to use

Reply:
electron supports the filechooser portal? Does it use that?